rclcpp/src/rclcpp/context.cpp

```diff
@@ -219,7 +219,7 @@ Context::init(
     if (0u == count) {
       ret = rcl_logging_configure_with_output_handler(
         &rcl_context_->global_arguments,
-        rcl_init_options_get_allocator(init_options_.get_rcl_init_options()),
+        rcl_init_options_get_allocator(init_options.get_rcl_init_options()),
```

Comment:
I believe this is just a bug, `init_options_` is not initialized yet here, it should use `init_options` from argument.